diff --git a/js/3rdparty/webodf/editor/MemberListView.js b/js/3rdparty/webodf/editor/MemberListView.js
index cbab8ec..a6381aa 100644
--- a/js/3rdparty/webodf/editor/MemberListView.js
+++ b/js/3rdparty/webodf/editor/MemberListView.js
@@ -73,6 +73,10 @@ define("webodf/editor/MemberListView",
                             node.src = memberDetails.imageurl;
                             // update border color
                             node.style.borderColor = memberDetails.color;
+						} else if (node.localName === "span" && memberDetails.imageurl){
+							$(node).avatar(memberDetails.imageurl, 60);
+							node.style.borderColor = memberDetails.color;
                         } else if (node.localName === "div") {
                             node.setAttribute('fullname', memberDetails.fullname);
                         }
@@ -92,7 +96,7 @@ define("webodf/editor/MemberListView",
             var doc = memberListDiv.ownerDocument,
                 htmlns = doc.documentElement.namespaceURI,
                 avatarDiv = doc.createElementNS(htmlns, "div"),
-                imageElement = doc.createElement("img"),
+                imageElement = doc.createElement("span"),
                 fullnameNode = doc.createElement("div");
 
             avatarDiv.className = "memberListButton";
