diff --git b/js/3rdparty/webodf/editor/MemberListView.js a/js/3rdparty/webodf/editor/MemberListView.js
index 83074ba..9f604c7 100644
--- b/js/3rdparty/webodf/editor/MemberListView.js
+++ a/js/3rdparty/webodf/editor/MemberListView.js
@@ -76,6 +76,11 @@ define("webodf/editor/MemberListView",
                             node.src = memberDetails.imageUrl;
                             // update border color
                             node.style.borderColor = memberDetails.color;
+						} else if (node.localName === "span" && memberDetails.imageUrl){
+							try {
+								$(node).avatar(memberDetails.imageUrl, 60);
+							} catch (e){}
+							node.style.borderColor = memberDetails.color;
                         } else if (node.localName === "div") {
                             node.setAttribute('fullname', memberDetails.fullName);
                         }
@@ -95,7 +100,7 @@ define("webodf/editor/MemberListView",
             var doc = memberListDiv.ownerDocument,
                 htmlns = doc.documentElement.namespaceURI,
                 avatarDiv = doc.createElementNS(htmlns, "div"),
-                imageElement = doc.createElement("img"),
+                imageElement = doc.createElement("span"),
                 fullnameNode = doc.createElement("div");
 
             avatarDiv.className = "memberListButton";
@@ -113,7 +118,7 @@ define("webodf/editor/MemberListView",
             avatarDiv.onclick = function () {
                 var caret = editorSession.sessionView.getCaret(memberId);
                 if (caret) {
-                    caret.toggleHandleVisibility();
+                    //caret.toggleHandleVisibility();
                 }
             };
             memberListDiv.appendChild(avatarDiv);
